### DISCUSSION TITLE
Secure transfer to Windows

### DISCUSSION THREAD

**Discussion by ATBenoit** [aonez/Keka#1402] [General]
Greetings... I am wondering if Keka is the software to solve my challenge. I need to send a large number of files securely over the web to someone who uses a Windows computer. Does the person at the other end need Keka to unlock and decompress the sent files?

**Comment by aonez** [comment]
The quick answer is no.

But keep in mind depending on the format you use the other person will need a third party application such as [7-Zip](https://www.7-zip.org).

- If you compress using the ZIP format (without AES-256) it will be compatible with all operating systems out of the box
- If you use ZIP + AES-256 (more secure) or 7Z (more compression + secure) it may need a third party utility

Hope this helps.

**Reply by ATBenoit** [reply]
Understood. Thank you.

**Comment by rshackelf0rd** [comment]
> But keep in mind depending on the format you use the other person will need a third party application such as [7-Zip](https://www.7-zip.org).

If I'm using Keka, is there any format that wouldn't require a Windows user to need a third party app?

I have to provide project files and assets to a client in <4gb ZIPs. My current process is rather time consuming. Being able to split a ZIP with something like Keka would be a huge time saver for me, but I can't rely on them having third party apps.

**Reply by aonez** [reply]
I'm afraid Windows does not support volume files without a third party utility. ZIP is universally supported, but not if splitted. TAR, GZIP and BZIP2 are also supported but again not if splitted.

**Reply by aonez** [reply]
I'll implement the option to create self extracting 7Z files for Windows, that way you can compress, encrypt and split files using 7Z and the Windows user does not need to download or install a third party utility. I'll get back to you.

**Comment by aonez** [comment]
Already did a version you both can test that can create self extracting 7Z and RAR files, so Windows users can benefit from the better format and encryption methods without having to install a third party utility.

You can download the preview version here:
https://github.com/aonez/Keka/releases/download/dev-test-builds/Keka-v1.4.0-dev.r5348.7z

Just enable the "Self extracting archive for Windows". Note this is a preview version, although it should be pretty safe to use it as your main version. This change will come to the stable version (Web and App Store) later.

<img width="448" alt="Screenshot 2024-03-01 at 18 31 05" src="https://github.com/aonez/Keka/assets/480163/4502cc8c-8b1d-4d50-ac85-481a549db234">
